oemoflex/tools/labels.yaml

```diff
@@ -9,4 +9,17 @@
 !!python/tuple ['BB-electricity-shortage', 'BB-electricity', 'flow']: Shortage
 !!python/tuple ['BB-solar-pv', 'BB-electricity', 'flow']: PV
 !!python/tuple ['BB-wind-onshore', 'BB-electricity', 'flow']: Wind
-!!python/tuple ['BE-BB', 'BB-electricity', 'flow']: Import
\ No newline at end of file
+!!python/tuple ['BE-BB', 'BB-electricity', 'flow']: Import
+
```

Comment:
Why is there an empty line here?

Reply:
the upper part contains the headers of BB-electrcity.csv and the lower part the ones of BE-electricity.csv
I thought it is better to distinguish between both by putting an empty line there.
Or should I rather remove it?

Reply:
Oh, I guess we will have to rethink that. I did not see that before - the regions are hard-coded, which they should not be. oemoflex does not know about the names of the regions.

Reply:
Ah right, so in this case we need to avoid "BB" and "BE" in the strings but the rest is universal?
I think about how to solve that.

Reply:
Exactly. Thanks!

Reply:
We can get the region name from the argument bus_name. But I don't know how to deal with the electricity flows import and export. Do you have any idea?